src/edu/washington/escience/myria/CsvTupleWriter.java

```diff
@@ -58,6 +61,11 @@ private CsvTupleWriter(final OutputStream out, final CSVFormat csvFormat) throws
   }
 
   @Override
+  public void open(final OutputStream stream) throws IOException {
```

Comment:
Hey @jingjingwang, I the latest commit here does what Tobin suggested where we remove the old constructors and use the new open methods. I think it looks cleaner as well. Let us know if you also agree when you get the chance.